src/robotide/parserlog/parserlog.py

```diff
@@ -19,12 +19,6 @@
 import atexit
 import glob
 import sys
-try:
```

Comment:
This import is important to be checked if in wxPython < 3.0.3. We should keep it for now.

Reply:
I understand, but the import is not used in this file. There is no point in keeping it.

Reply:
OK. I think I left it there for future use.